website/docs/r/elasticsearch_domain.html.markdown

```diff
@@ -224,6 +224,7 @@ The following arguments are supported:
 * `vpc_options` - (Optional) VPC related options, see below. Adding or removing this configuration forces a new resource ([documentation](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-vpc.html#es-vpc-limitations)).
 * `log_publishing_options` - (Optional) Options for publishing slow logs to CloudWatch Logs.
 * `elasticsearch_version` - (Optional) The version of Elasticsearch to deploy. Defaults to `1.5`
+* `enforce_httpsnt_options` - (Optional) Domain endpoint HTTP(S) related options. See below.
```

Comment:
Looks like this fix got transposed between here and line 246 -- will fix on merge. 👍